Source/Libraries/CorruptCore/Stockpile/Stockpile.cs

```diff
@@ -46,7 +44,7 @@ public Stockpile()
 
         public override string ToString()
         {
-            return Name ?? string.Empty;
+            return string.Empty;
```

Comment:
While Name was unused, it was there to go with the pattern of blastunit & stashkey having a Name & overridden ToString() that returns their name when called. Please revert this one.

Reply:
Done